HybrasylXML/XSD/hybrasylExtensions.cs

```diff
@@ -173,16 +184,63 @@ public int Id
 
         public byte CastableLevel { get; set; }
 
+        public DateTime LastCast { get; set; }
+
+        public bool OnCooldown
+        {
+            get
+            {
+                if (Cooldown > 0)
+                    return (DateTime.Now - LastCast).Seconds > Cooldown;
```

Comment:
Should this be a less-than sign instead? If number of seconds passed is LT the cooldown, then false....

Reply:
Fixed